### PR TITLE
Milestone3: contact form section

### PR DESCRIPTION
**Pull request by RuriEnciso**
### **Milestone3: contact form section**

In this section we: 
- Added contact form section
- Added email requierements
- Added name requirements
- Added the Footer section.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -16,6 +16,10 @@ body {
   margin: 0 auto;
 }
 
+/* header {
+  position: fixed;
+} */
+
```
Comment (ademibowale) [inline style.css]: - [ ] I would suggest you to kindly remove the commented code linke commented code line makes code look messy and misleading 
- [ ] Also i would suggest you kindly remove all commented code line in your readme

File: index.html [hunk 2]
```diff
@@ -176,7 +176,31 @@ <h3 class="tool-title">Skills</h3>
...
+            <p class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form action="https://formspree.io/f/xqkonber" method="POST">
+                <label for="name"></label>
+                <input type="text" id="name" name="name" maxlength="30" class="imput-contact" placeholder="Full name" required>
+                <label for="mail"></label>
+                <input type="email" id="mail" name="email" class="imput-contact" placeholder="Email address" required>
+                <label for="msj"></label>
+                <textarea name="message" rows="5" maxlength="500" id="msj" placeholder="Hey I’d like to connect to..." required></textarea>
+                <button class="see-project" type="submit">Get in touch</button>
+            </form>
```
Comment (ademibowale) [inline index.html:191]: - [ ] You have done so much great work on this project we are proud of you but there are ssomething you still need to address please kindly work more  on (Get in touch button) Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.Its very important to stick to figma design template to make your website look great please kindly fix this.

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @RuriEnciso , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the required attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Pair-programming collaborative activity ✔️
- Good use of Github flow ✔️
- All linters check passed

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @RuriEnciso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**